### DISCUSSION TITLE
为什么loadEnv后，要给process.env赋值后，import.meta.env里才能取值？

### DISCUSSION THREAD

**Discussion by madawc** [alex8088/electron-vite#410] [Q&A]
`

export default defineConfig(({ command, mode }) => {
  const env = loadEnv('', process.cwd(), ['MYENV'])
  for (const key in env) {
    console.log(key + '=' + env[key])
    process.env[key] = env[key]
  }
`

如果我把上面代码中的process.env[key]=env[key]去掉，那么后续组件中无法通过import.meta.env.XXXX取值。
如果不去掉，就能通过import.meta.env.xxxx取值。
像问下，这个是什么原理？
另外，要调试defineConfig方法的话，要怎么调？用的vscdoe开发。
刚学electron，一堆不明白的问题。

**Comment by alex8088** [comment]
import.meta.env 与 process.env 无关， process.env 属于nodejs环境，没有更明确的信息不了解问题所在。请确认是正确使用

https://vitejs.dev/guide/env-and-mode.html

https://electron-vite.org/guide/env-and-mode
